cmd/send-email/main.go

```diff
@@ -25,7 +28,18 @@ func main() {
         log.Fatal("Could not load config options.", err);
     }
 
-    err = email.Send(args.To, args.Subject, args.Body, false);
+    // If database cannot get course and using role based addressing
```

Comment:
Generally, we want comment to be complete sentences (with proper punctuation and casing).

Reply:
Unresolved.

This is still an issue in other comments.